cmd/workload-controller-manager/main.go

```diff
@@ -92,7 +92,7 @@ func getConfig() (*controllerManagerConfig.Config, error) {
 	}
 
 	klog.Infof("Current workload controller port %d", workloadControllerPort)
-	klog.Infof("Rest client QPS %v, kubeconfig [%#v]", controllerManagerKubeConfig.GetConfig().QPS, controllerManagerKubeConfig.GetConfig())
```

Comment:
Nit: I see this bug is about QPS. But for the purpose of information, we need the controllerManagerKubeConfig.GetConfig(), not just QPS.

Reply:
GetConfig has a lot of information. including cert data. It is best to print out non sensitive and useful information.